resources/js/components/global/Navbar.vue

```diff
@@ -10,27 +10,37 @@
   <ul id="dropdown" class="navbar__item-wrapper" :class="isNavOpen ? 'navbar__item-wrapper--active' : ''">
     <li class="navbar__item">
       <router-link v-show="!isAdminUser" to="/" class="navbar__link">
-        Dashboard
+        <div>
```

Comment:
These  `<div>`s are unnecessary. The CSS properties you added to `.navbar__link` is all you need.